### DISCUSSION TITLE
Customization

### DISCUSSION THREAD

**Discussion by premy-punchu** [austintoddj/canvas#1399] [General]
How will I be able to customize the template

**Comment by austintoddj** [comment]
If you followed [the instructions here](https://github.com/austintoddj/canvas?tab=readme-ov-file#canvas-ui), you can edit the interface in:
- resources/js/canvas-ui
- resources/sass/canvas-ui

If you want to see how all the files are moved from stubs to the appropriate views, controllers, etc - check out the [Console Command here](https://github.com/austintoddj/canvas/blob/master/src/Console/UiCommand.php).

**Reply by premy-punchu** [reply]
Can I add bootstrap templates then ?

**Reply by austintoddj** [reply]
Yes, you can.
